sklearn/utils/testing.py

```diff
@@ -697,5 +706,34 @@ def check_skip_travis():
     if os.environ.get('TRAVIS') == "true":
         raise SkipTest("This test needs to be skipped on Travis")
 
+
+def delete_folder(folder_path):
```

Comment:
This is a private utility not meant to be used directly, only via the `TempMemmap` context manager. Therefore I would rename it as `_delete_folder`.